packages/amesos2/src/Amesos2_EpetraRowMatrix_AbstractMatrixAdapter_decl.hpp

```diff
@@ -151,6 +151,11 @@ namespace Amesos2 {
     const RCP<const Tpetra::Map<local_ordinal_t,
 				global_ordinal_t,
 				node_t> >
+    getMap_impl() const;
```

Comment:
Yes, I needed to add this in order to get the gatherMap call to behave as desired